src/AotPackageReference/project.lock.json

```diff
@@ -1,16 +1,19 @@
 {
-  "locked": true,
-  "version": -9996,
+  "locked": false,
```

Comment:
I don't understand. I believe this was auto-generated, do you want me to edit it?